react/components/form/BaseInput.js

```diff
@@ -39,17 +39,17 @@ const BaseInput = props => {
       error={!!currentError}
       errorMessage={currentError}
       helpText={schema.description}
-      label={<FormattedMessage id={label}/>}
-      max={max}
-      min={min}
+      label={label}
+      max={max ? `${max}` : max}
```

Comment:
I think it'd be better to just use `` max && `${max}` ``, as the fallback value is already `undefined` (that also applies to the other changes)

Reply:
Done!